automatedTestChrome.py

```diff
@@ -0,0 +1,59 @@
+from helium.api import *
+import time
+
+
+def logout():
+    # used to log out normal user to test superuser
+    click("Logout")
+
+def superUserTest():
+
+    # logs in superuser
+    go_to("http://73.229.199.171/")
+    write("god", into="Username")
```

Comment:
We should store these credentials as environment variables or as a `.config` file that we don't track

Reply:
ok ill change them to environment variables